src/components/Viewport.ts

```diff
@@ -384,11 +384,26 @@ export default class Viewport {
   }
 
   public enable(): void {
-    this.panInput.enable();
+    if (!this.panInput) {
+      const options = this.options;
+      this.panInput = new PanInput(this.viewportElement, {
+        inputType: options.inputType,
+        thresholdAngle: options.thresholdAngle,
+        iOSEdgeSwipeThreshold: options.iOSEdgeSwipeThreshold,
+        scale: options.horizontal ? [-1, 0] : [0, -1],
+      });
+
+      this.axes.connect(options.horizontal ? ["flick", ""] : ["", "flick"], this.panInput);
+    }
   }
 
   public disable(): void {
-    this.panInput.disable();
+    if (this.panInput) {
+      this.panInput.destroy();
+      this.panInput = null;
```

Comment:
Is the original paninput disconnected?

Reply:
Yes it calls `disconnect()` on destroy.
![image](https://user-images.githubusercontent.com/26213435/104685667-ec5c6880-573e-11eb-85ab-1c4cf1286c92.png)

Reply:
@WoodNeck 

I mean, it is said to `axes.disconnect`

Reply:
Looks like `axes.disconnect` also calls `disconnect` of a inputType, so I guess it's okay?
![image](https://user-images.githubusercontent.com/26213435/104686021-ba97d180-573f-11eb-8200-119fde4f1af6.png)

Reply:
Um... it needs to call the destroy method of paninput and then the disconnect method.

Reply:
Why should it have to be?

Reply:
Well. Should the axes be connected, shouldn't I disconnect? Even if the axes don't call disconnect, there seems to be no problem.

Reply:
I mean, as the PanInput calls `disconnect()` while calling `destroy()` why should it have to be called again?